src/manifest.json

```diff
@@ -19,7 +19,7 @@
 		{
 			"run_at": "document_start",
 			"matches": [
-				"https://*.reddit.com/*"
+				"https://www.reddit.com/*"
```

Comment:
Oops, forgot about that subdomain, I'll add it back. This change was to prevent the extension from running on old.reddit.com, plus all of the 2 letter language code subdomains (eg. np.reddit.com).

Reply:
That makes sense. I think it should be okay to just specify www and new here instead of the wildcard; I didn't initially realize that it's not possible to get new Reddit on the language subdomains, otherwise I probably would've just done it that way before.